guides/common/assembly_monitoring-hosts-using-red-hat-insights.adoc

```diff
@@ -4,6 +4,14 @@ ifdef::orcharhino[]
 include::modules/proc_installing-red-hat-cloud-plug-in.adoc[leveloffset=+1]
 endif::[]
 
-include::modules/proc_using-red-hat-insights.adoc[leveloffset=+1]
+include::modules/ref_access-to-information-from-insights-in-project.adoc[leveloffset=+1]
+
+include::modules/proc_excluding-hosts-from-rh-cloud-and-insights-client-reports.adoc[leveloffset=+1]
+
+include::modules/proc_deploying-red-hat-insights-using-the-ansible-role.adoc[leveloffset=+1]
+
+include::modules/proc_configuring-automatic-synchronization-of-insights-recommendations-for-hosts.adoc[leveloffset=+1]
+
+include::modules/proc_configuring-automatic-removal-of-hosts-from-the-insights-inventory.adoc[leveloffset=+1]
 
 include::modules/proc_creating-an-insights-plan-for-hosts.adoc[leveloffset=+1]
```

Comment:
Could we rename this module to "Creating an Insights _Remediation_ Plan for Hosts", please? :)

Reply:
Addressed :+1: